### DISCUSSION TITLE
Support of vite config file and spa sources in a subfolder

### DISCUSSION THREAD

**Discussion by flotoz** [Eptagone/Vite.AspNetCore#30] [Ideas]
Hello,

I'm just started to use your nuget package, great job!
But my vite config file is in a subfolder, and it's no way to configure it for my need. 
is it possible to officially add support of specific folder for vite config for a future version?

I've just replaced:
` var workingDirectory = environment.ContentRootPath;`
by
` var workingDirectory = Path.Combine(environment.ContentRootPath, this._viteOptions.Server.Path);`

and add the field into ViteServerOptions.cs
` 
    /// <summary>
    /// The path to the vite config file. Default value is the current directory.
    /// </summary>
    public string Path { get; init; } = "/";
`

**and every thing is working fine for me.**

Thank you in advance,
Florian METOZ

**Comment by Eptagone** [comment]
Hi, so you want to have your `package.json` in a subfolder, right?
Yes, i think it's possible to add that feature.

**Reply by flotoz** [reply]
Yes exactly.

**Reply by Eptagone** [reply]
Okey, I'll add it with the next release

**Comment by Eptagone** [comment]
This feature will be in the v1.8 of the package.
